src/components/CreateNewProjectWizard/CreateNewProjectWizard.js

```diff
@@ -197,7 +197,7 @@ class CreateNewProjectWizard extends PureComponent<Props, State> {
 
 const mapStateToProps = state => ({
   projects: getById(state),
-  projectHomePath: getProjectHomePath(state.paths),
+  projectHomePath: getProjectHomePath(state),
```

Comment:
It's good form for selectors to take the entire `state` object, not just a slice of it... that way every selector behaves identically.

Reply:
Good call. It would be nice to type them tho: right now the global state is always `any` IIRC

Reply:
yeah, I believe there's an issue for this. It's kind of a big thing, but agree it would be really valuable